### PR TITLE
adds confirm edit checkmark and slider handle, closes #1195 

### LINKED ISSUE #1195: Add confirm edit checkmark - and slider handle
**Describe the bug**
My partner was confused on how to close the edit menu.  

**To Reproduce**
1. Go to 'My Goals'
2. Click on floating +
3. Enter text  
4. Ask the person how to close the edit menu

**Expected behavior**
It's intuitive how to close the menu when there is a big checkmark:  
image

After a while you will mis-click and realize you can click anywhere to close - not just on the checkmark.  

As a second option there should be a slider handle that indicates you can slide the bottom drawer down:  
image

**Screenshots**
image

**Additional context**
This was present before, but not as a floating button.  
A floating button is preferred - as the previous option took away screen space from the edit title component.

### PR DESCRIPTION
**Pull request by alpeshl**
adds confirm edit checkmark and slider handle, closes #1195

### CODE REVIEW

**Comment by tijlleenders** [comment]
Looking good @alpeshl !

I'm having trouble using the slider to close though.
@Tushar-4781 If that is tricky to implement we can merge this for now as the checkmark already has value - and make a different issue to implement slider?

**Comment by Tushar-4781** [comment]
> Looking good @alpeshl !
> 
> I'm having trouble using the slider to close though. @Tushar-4781 If that is tricky to implement we can merge this for now as the checkmark already has value - and make a different issue to implement slider?

@tijlleenders  The slider won't work because it just a line. I think @alpeshl misunderstood the context of slider. @alpeshl try using drag properties of html element to pull the slider for closing the pop up.

**Comment by alpeshl** [comment]
> > Looking good @alpeshl !
> > I'm having trouble using the slider to close though. @Tushar-4781 If that is tricky to implement we can merge this for now as the checkmark already has value - and make a different issue to implement slider?
> 
> @tijlleenders The slider won't work because it just a line. I think @alpeshl misunderstood the context of slider. @alpeshl try using drag properties of html element to pull the slider for closing the pop up.

For now, I've added logic for slider handle to close the popup by implementing touch move gesture event. 

Since, we're using Modal component from `react-bootstrap`, it is not really easy to implement drag support and customize the existing behavior as expected.

Please check the update and let me know in case of any issue or feedback.

**Comment by Tushar-4781** [comment]
@alpeshl  Nicely done. Maybe by using margin or padding props, you can increase the tap area of the slider. Then it will work like charm.

However there is an issue to convert all modals into antd modals. So don't worry about that.

**Comment by alpeshl** [comment]
> @alpeshl Nicely done. Maybe by using margin or padding props, you can increase the tap area of the slider. Then it will work like charm.
> 
> However there is an issue to convert all modals into antd modals. So don't worry about that.

done

**Comment by Tushar-4781** [comment]
@alpeshl please make the requested changes as well

**Comment by alpeshl** [comment]
> @alpeshl please make the requested changes as well

I already pushed requested changes, could you please review and verify.

**Comment by alpeshl** [comment]
@tijlleenders @Tushar-4781 can this be merged?

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]

File: src/components/GoalsComponents/GoalConfigModal/GoalConfigModal.tsx [hunk 1]
```diff
@@ -232,7 +233,39 @@ const GoalConfigModal = ({ goal }: { goal : GoalItem }) => {
...
+          >
+            <img
+              alt="Save Goal"
+              src={correct}
+              style={{ cursor: "Pointer" }}
+              className={`${darkModeStatus ? "dark-svg" : ""}`}
+            />
+          </button>
+          <div
+            style={{ display: "flex",
```
[the remark this answers is not in the document]
Comment (alpeshl) [reply src/components/GoalsComponents/GoalConfigModal/GoalConfigModal.tsx]: Slider seems to be working fine. I've moved styles to scss module.

Let me know if you find any issue.

**Review by Tushar-4781 — APPROVED** [review]

**Pull request merged**